docs/get-started.md

```diff
@@ -0,0 +1,237 @@
+  <details>
+    <summary><strong>Expand for setup instructions</strong></summary>
+    To create your first Elastic Observability deployment:
+
+    1. Sign up for a [free Elastic Cloud trial](https://cloud.elastic.co/registration) or sign into an existing account.
+    1. Go to <https://cloud.elastic.co/home>.
+    1. Click **Create deployment**.
+    1. When the deployment is ready, click *Open* to visit your Kibana home page (for example, `https://{DEPLOYMENT_NAME}.kb.{REGION}.cloud.es.io/app/home#/getting_started`).
+  </details>
+
+<!-- ✅ How to install EDOT .NET -->
+## Install
+
+<!-- ✅ Step-by-step instructions -->
+To get started with the Elastic OpenTelemetry Distribution for .NET, add the `Elastic.OpenTelemetry` NuGet package reference to your project file:
+
+```xml
+<PackageReference Include="Elastic.OpenTelemetry" Version="<LATEST>" />
+```
+
+> [!NOTE]
+> Replace the `<LATEST>` placeholder with the latest available package from [NuGet.org](https://www.nuget.org/packages/Elastic.OpenTelemetry).
+
+After adding the package reference, you can start using EDOT .NET in your application.
+
+<!-- ✅ Any additional info related to installation -->
+> [!NOTE]
+> EDOT .NET includes a transitive dependency on the OpenTelemetry SDK, so you do not _need_ to add the OpenTelemetry SDK package to your project directly. However, you _can_ explicitly add the OpenTelemetry SDK as a dependency if you want to opt into newer SDK versions. If you do this, the OpenTelemetry SDK dependency must be defined _before_ the Elastic OpenTelemetry Distribution for .NET is defined.
+
+> [!NOTE]
+> EDOT .NET is designed to be easy to use and integrate into your applications. This includes applications that have previously used the OpenTelemetry SDK directly. In situations where the OpenTelemetry SDK is already used, the only required change is to add the `Elastic.OpenTelemetry` NuGet package to the project. Doing so will automatically switch to the opinionated configuration provided by EDOT .NET.
+
+<!-- ✅ Start-to-finish operation -->
+## Instrument your application
+
+The OpenTelemetry SDK and EDOT .NET provide extension methods to enable observability features in your application with a few lines of code. Choose from:
+
+* [**ASP.NET Core application**](#aspnet-core-application) for ASP.NET Core minimal API applications, other ASP.NET Core workloads and other host-based applications like [worker services](https://learn.microsoft.com/en-us/dotnet/core/extensions/workers).
+* [**`Microsoft.Extensions.Hosting`**](#microsoftextensionshosting) for console applications and services that are written against a builder that exposes an `IServiceCollection`.
+* [**Manual instrumentation**](#manual-instrumentation) for environments where an `IServiceCollection` is unavailable.
```

Comment:
Does this framing make sense? Since we're keeping the content in GitHub (not publishing to the Elastic docs site), I attempted to combine the existing README content with the content in the docs directory.

Reply:
This works for now I'll reword these bullet points in a follow up PR (and ask for your review instead :)).

I need to add more content around `Automatic Instrumentation` that recently landed as a feature.

Reply:
Sounds good!